test/test_coverage.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+# Create a link for the code to current dir
+ln -s src/main/scala/org ./
```

Comment:
Is this the only way to make `sbt coveralls` work?  They should have some configurable settings.  I guess you probably tried everything, so let's write a clear comment **why** we do this business, not what it's doing. These comments aren't useful because they're longer than the actual commands and only repeat what they do.  Also, you probably want to say something about the required sbt and scala versions, here or in the build files or somewhere in test scripts.